awesome-books-with-ES6/index.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-undef */
+/* eslint-disable import/no-cycle */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Kindly enable linters here
- [ ] Let's try to maintain the YAGNI principle. Right now, the index.js file in the root of your project directory is useless. You should probably delete the `./index.js` file since it is no longer used in your app.
- [ ] **[OPTIONAL]** It would be nice move this file to the root path so users can easily access it.